tests/ui/user/addUser.negative.test.ts

```diff
@@ -0,0 +1,87 @@
+import { test, expect } from '@playwright/test';
+import AddUserPage from '../../../pages/ui/addUserPage';
+import { faker } from '@faker-js/faker';
+import { Gender } from '../../../enums/gender.enum';
+import { UserDTO } from '../../../dto/userDTO';
+
+let addUserPage: AddUserPage;
+
+const shortName: UserDTO = {
+  gender: Gender.Female,
+  name: faker.string.alpha({ length: 2 }),
+  yearOfBirth: 2001
+};
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('Forms/AddUser');
```

Comment:
Won't it be better to move this URL to the constants file? 
You use it at least twice in your code in different places and it will be better to have one generic variable for this